projects/observability/src/pages/explorer/explorer.component.ts

```diff
@@ -146,7 +154,16 @@ export class ExplorerComponent {
     this.vizDashboard$ = this.explorerDashboardBuilder.visualizationDashboard$;
     this.initialState$ = activatedRoute.queryParamMap.pipe(
       take(1),
-      map(paramMap => this.mapToInitialState(paramMap))
+      map(paramMap => this.mapToInitialState(paramMap)),
+      tap(initialState => {
```

Comment:
we can do this without tap, please just use distinct observables.

Reply:
Done!!